students/saleta_sebastian/lesson_05_lists/numbers_filter.py

```diff
@@ -0,0 +1,18 @@
+def filter(list, filter_range):
```

Comment:
filter and list are built-in functions. Try to avoid such names as: https://docs.python.org/3/library/functions.html